src/test/resources/features/refapp-2.x/openConceptLab/openConceptLab.feature

```diff
@@ -0,0 +1,19 @@
+Feature: OCL Subscription module
+ 
+Background:
+    Given User clicks on configure Metadata link from home page
+    When User selects Manage OCL
+    Then System loads Open Concept Lab page
+ 
+  @selenium
+  @subscription
+  Scenario: OCL MANAGEMENT
+     And User clicks on Setup subscription button
+     Then System on Subscription page
+      And  the user enters the URL of a new released dictionary
+     Then the "Token" title is visible
+      And the user enters the "User url
+      And User clicks on the Save Changes button
+     Then the user should be on the "Open Concept Lab" page
```

Comment:
Ii think after saving the changes, the user doesnt have to go this step but rather the api should be shown the previous imports just as the next step you have written.